source/common/http/conn_manager_impl.cc

```diff
@@ -821,6 +819,27 @@ void ConnectionManagerImpl::ActiveStream::refreshCachedRoute() {
   cached_route_ = std::move(route);
 }
 
+void ConnectionManagerImpl::ActiveStream::sendLocalReply(
+    bool is_grpc_request, Code code, const std::string& body,
+    std::function<void(HeaderMap& headers)> modify_headers) {
+  Utility::sendLocalReply(is_grpc_request,
+                          [this, modify_headers](HeaderMapPtr&& headers, bool end_stream) -> void {
+                            if (headers != nullptr && modify_headers != nullptr) {
```

Comment:
same comment can headers be nullptr? I think not?

Reply:
Right, it can't, removed the test for non-null.